docs/quick-start-guide.md

```diff
@@ -50,6 +50,41 @@ $ kubectl describe deployment sparkoperator -n sparkoperator
 
 ```
 
+### Metrics
+
+The operator exposes a set of metrics via the metric endpoint to be scraped by `Prometheus`. This is disabled by default and can be enabled by setting  the flag `-enable-metrics=true` and running 
+ `kubectl apply -f manifest/spark-operator.yaml`. You will also need to add the prometheus annotations `prometheus.io/scrape`
+  and `prometheus.io/port` which are used by Prometheus to scrape the metric endpoint if you are enabling metrics.
+  
+If enabled, the operator generates the following metrics:
```

Comment:
I suggest we use a table that explains what each metric is for.

Reply:
makes sense, done